nearlib/account.js

```diff
@@ -151,5 +151,40 @@ class Account {
     async viewAccount (accountId) {
         return await this.nearClient.viewAccount(accountId);
     }
+
+    /**
+     * Returns full details for a given account.
+     * @param {string} accountId id of the account to look up 
+     */
+    async getAccountDetails(accountId) {
+        accountId; // make linter happy
+        // Hardcoded for now
+        const result = {
+            authorizedApps : [
```

Comment:
these are called access keys in other places if I understand correctly:
https://github.com/nearprotocol/nearcore/pull/918/files

it also uses `amount` instead  of `balance` and doesn't even have `name` from what I see there

Reply:
transactions below also use `amount`

Reply:
Name - We do need an app "name" though, and ideally nearlib would provide that to UI directly without UI having to collect data from various places. We'll want to eventually add some app metadata to the system to enable this. 
Amount instead of balance - done
I am not 100% sold on renaming authorizedApps to access keys - accessKeys as a name seems too much coupled with implementation.